elements/pfelement/src/pfelement.js

```diff
@@ -39,8 +39,8 @@ class PFElement extends HTMLElement {
     return this.querySelector(`[slot='${name}']`);
   }
 
-  has_slot(name) {
-    return this.querySelector(`[slot='${name}']`);
+  has_slots(name) {
+    return this.querySelectorAll(`[slot='${name}']`);
```

Comment:
Should we go ahead and make this an array for convenience? `return [...this.querySelectorAll(`[slot='${name}']`)];`

Reply:
@castastrophe Let's go with `[...this.querySelector(`

Reply:
Sorry just to clarify - `[...this.querySelectorAll`?